### PR TITLE
Explain how to use grcov with Travis

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,32 @@ genhtml -o report/ --show-details --highlight --ignore-errors source --legend lc
 grcov ~/Documents/FD/mozilla-central/build -t coveralls -s ~/Documents/FD/mozilla-central --token YOUR_COVERALLS_TOKEN > coveralls.json
 ```
 
+### GRCOV with Travis
```
Comment (marco-c) [inline README.md]: Nit: `grcov` in lowercase

File: README.md [hunk 2]
```diff
@@ -54,6 +54,32 @@ genhtml -o report/ --show-details --highlight --ignore-errors source --legend lc
...
+
+Here is an example of .travis.yml file
+```YAML
+sudo: false
```
Comment (marco-c) [inline README.md]: `sudo` is no longer needed

File: README.md [hunk 3]
```diff
@@ -54,6 +54,32 @@ genhtml -o report/ --show-details --highlight --ignore-errors source --legend lc
...
+  - curl -L https://github.com/mozilla/grcov/releases/download/v0.4.1/grcov-linux-x86_64.tar.bz2 | tar jxf -
+
+matrix:
+  include:
+    - os: linux
+      rust: nightly
+
+script:
+    - export CARGO_INCREMENTAL=0
+    - export RUSTFLAGS="-Zprofile -Ccodegen-units=1"
```
Comment (marco-c) [inline README.md]: We could suggest disabling inlining too (-Cinline-threshold=0), and maybe forcing the linking of dead code (-Clink-dead-code) and disabling overflow checks (-Coverflow-checks=off).

We are not doing these things ourselves, maybe we should :)

File: README.md [hunk 4]
```diff
@@ -54,6 +54,32 @@ genhtml -o report/ --show-details --highlight --ignore-errors source --legend lc
...
+    - cargo build --verbose $CARGO_OPTIONS
+    - cargo test --verbose $CARGO_OPTIONS
+    - |
+      zip -0 ccov.zip `find . \( -name "YOUR_PROJECT_NAME*.gc*" \) -print`;
```
Comment (marco-c) [inline README.md]: What if we remove `YOUR_PROJECT_NAME*` and just find all files ending with gcda or gcno?

Or maybe we can just execute grcov on `.`, without zipping anything.

Comment (calixteman) [reply README.md]: there are a lot of gcno, gcda for dependencies we don't have source files in the repo so it's useless to have them.

**Pull request merged**